.gitignore

```diff
@@ -1,16 +1,44 @@
+# yarn
 .yarn/*
+.yarn/cache
+.yarn/install-state.gz
 !.yarn/patches
 !.yarn/plugins
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
-coverage
-**/.DS_Store
 
 # Swap the comments on the following lines if you don't wish to use zero-installs
 # Documentation here: https://yarnpkg.com/features/zero-installs
 #!.yarn/cache
 .pnp.*
 
-node_modules
-.idea
+# modules
+node_modules/
+.docusaurus/
+.idea/
+
+# testing
+coverage/
+
+# production
+build/
```

Comment:
please don't do that. This line is indicating to github that we would like to ignore ALL directories called `build` in the repository. This is not a good practice and this is not what we want. If you want to ignore a specific build directory, use the gitignore file located in the NPM sub project

Reply:
Done